code/controllers/CacheableCacheInspectorController.php

```diff
@@ -124,9 +136,14 @@ private function getTotalCachedObjects($stage, $className = 'CacheableSiteTree')
         if($cache && $siteMap = $cache->get_site_map()) {
             // For reasons unknown, items appear in object-cache with NULL properties
             $cachedSiteTree = ArrayList::create($siteMap)->filterByCallback(function($item) {
-                return !is_null($item->ParentID);
+                if(!is_null($item->ParentID)) {
+                    // push the item ID to the _cachedIDs array for comparison later
+                    $this->_cachedIDs[] = $item->ID;
```

Comment:
I haven't run the code in PHP's "dev mode" yet (E_ALL) but I suspect using `$this` in that context is illegal because `$this` isn't in the scope of the lambda function, you'd usually pass that in to PHP's `use` function.